contracts/contracts/strategies/BaseCompoundStrategy.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity ^0.8.0;
+
+/**
+ * @title OUSD Base Compound Abstract Strategy
+ * @author Origin Protocol Inc
+ */
+
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { ICERC20 } from "./ICompound.sol";
+import { IComptroller } from "../interfaces/IComptroller.sol";
+import { IERC20, InitializableAbstractStrategy } from "../utils/InitializableAbstractStrategy.sol";
+
+abstract contract BaseCompoundStrategy is InitializableAbstractStrategy {
```

Comment:
Agree.

Reply:
Good point thanks. Added in. 

Also if we decide to re-deploy CompoundStrategy this change won't cause any collisions since the CompoundStrategy contract doesn't only uses storage slots defined in `InitializableAbstractStrategy`